lib/workers/qemu.js

```diff
@@ -59,12 +70,21 @@ module.exports = class QEMUWorker {
       '-smp', '4'
     ])
 
-    childKillEventHandler(this.child, 'exit')
-    childKillEventHandler(this.child, 'uncaughtException')
-    childKillEventHandler(this.child, 'unhandledRejection')
+    this.child.stderr.on('data', (data) => {
+      stderr += data
+    })
+
+    this.child.on('exit', (code) => {
+      if (code !== 0) {
+        throw new Error(stderr)
+      }
+    })
+
+    manageHandlers(this.child, true)
   }
 
   async off () {
+    manageHandlers(this.child, false)
```

Comment:
Can we turn this into:

```js
manageHandlers(this.child, { register: false })
```

so we don't have to go looking for the definition of `manageHandlers` to figure out what the boolean means?